src/extensions/mountainview/MountainView/ViewContainer.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { FunctionComponent, useEffect, useState } from 'react';
```

Comment:
The ViewContainer is the right part of MountainView. It shows all the open views in a tab widget.